buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -52,9 +53,12 @@ class BuildPlugin implements Plugin<Project> {
 
     /** Returns a closure of common configuration shared by unit and integration tests. */
     static Closure commonTestConfig(Project project) {
+        def javaHome = System.getProperty("java.home")
+        if (!javaHome) {
```

Comment:
But falsy-ness! Also empty string would be pretty devastating here but sure, `== null` it is.